RELEASE_NOTES.md

```diff
@@ -360,10 +360,10 @@ changes (where available).
   lighttable in this scenario has now been drastically improved and is
   fast and responsive even with ~50k images and 20 thumbnails per line.
 
-- Fixed pixel errors in RAW Chromatic Aberration and LMMSE demosaic
+- Fixed pixel errors in raw chromatic aberrations module and LMMSE demosaic
```

Comment:
I'll keep the capitalized version. This is the release notes not the lower case interface.

Reply:
> I'll keep the capitalized version. This is the release notes not the lower case interface.

Then it should be 'Raw chromatic aberrations' and not the version in the text. And if we use the capitalized version here, we should do the same in other places where module names are mentioned. Like, 'RGB primaries' and 'Sigmoid'.

Reply:
We can choose any version, capitalized or not, but we must use it uniformly.